docs/exchange-operators/rosetta/samples/send-transactions.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Sending transactions
+description: Learn how to send transactions using a keypair
+---
+
+:::info
+This flow is also described in the [Construction API Overview](https://docs.cloud.coinbase.com/rosetta/docs/construction-api-overview) article in the official Rosetta documentation.
+:::
+
+The steps needed to send payment in MINA token are following:
+1. Given a keypair, derive the account indentifier using `/construction/derive` endpoint
```

Comment:
I won't mark the rest instances, but be sure to use two words for "key pair' 
https://github.com/o1-labs/docs2/wiki/Word-list#key-pair

Reply:
@barriebyron done